src/index.ts

```diff
@@ -27,6 +27,7 @@ export enum ProgressEvent {
 }
 
 export interface CallerOptions extends GoogleAuthOptions {
+  region?: string;
```

Comment:
I kinda wonder out loud if this should be optional.  For creating a function, I think having `us-central1` as a default makes sense.  For invoking the function - I think we should just require it, and throw if it isn't provided.

Reply:
_Thinking out loud_
I think most devs don't care where the region is.
For example, from Apps Script land, I have no clue where my code is running. I just want to run it.

I see either way, but I'm personally not a fan of throwing anything if a default is harmless.

Reply:
Oh cool - I didn't know that App Script kinda hard codes the region.  I'm cool with defaults 🤷‍♂️